dart/lib/src/sentry_stack_trace_factory.dart

```diff
@@ -101,14 +103,14 @@ class SentryStackTraceFactory {
 
     if (_inAppIncludes != null) {
       for (final include in _inAppIncludes) {
-        if (frame.package != null && frame.package.startsWith(include)) {
+        if (frame.package != null && frame.package == include) {
```

Comment:
as we get a package name I'm not sure if we should verify `startWith`

Reply:
that's what I said before, on Java it makes sense because we have packages.like.this and inner packages belong to the parent package